internal/service/enrollmentrequest.go

```diff
@@ -23,7 +23,7 @@ func validateAndCompleteEnrollmentRequest(enrollmentRequest *v1alpha1.Enrollment
 	if enrollmentRequest.Status == nil {
 		enrollmentRequest.Status = &v1alpha1.EnrollmentRequestStatus{
 			Certificate: nil,
-			Conditions:  &[]v1alpha1.Condition{},
+			Conditions:  []v1alpha1.Condition{},
```

Comment:
This is not valid because you required the fields in condition. I believe you need a default set of conditions?

Reply:
nmd read this wrong